backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs

```diff
@@ -25,13 +23,20 @@ public static void RegisterCustomServices(this IServiceCollection services)
                 .AddNewtonsoftJson(options => options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
 
             services.AddTransient<ISampleService, SampleService>();
+            services.AddTransient<ILessonsService, LessonsService>();
             services.AddScoped<IFirebaseAuthService, FirebaseAuthService>();
             services.AddFirebaseApp();
         }
 
         public static void AddAutoMapper(this IServiceCollection services)
         {
-            services.AddAutoMapper(Assembly.GetAssembly(typeof(SampleProfile)));
+            services.AddAutoMapper(
+                Assembly.GetAssembly(typeof(SampleProfile)),
+                Assembly.GetAssembly(typeof(LessonsProfile)),
+                Assembly.GetAssembly(typeof(UserProfile)),
+                Assembly.GetAssembly(typeof(TagForLessonProfile)),
+                Assembly.GetAssembly(typeof(QuestionsProfile)),
+                Assembly.GetAssembly(typeof(SubQuestionsProfile)));
```

Comment:
It is enough to declare only one of them

Reply:
fixed